content/guidance/inventory-code.md

```diff
@@ -13,4 +13,163 @@ sidenav: true
 sticky_sidenav: true
 ---
 
-Creating your enterprise code inventory
+# Creating your enterprise code inventory
```

Comment:
There are duplicate titles shown on this page, can remove this one! The title is already displayed as defined in front matter in line 2!